api/FetchEvent.json

```diff
@@ -432,6 +432,38 @@
             "deprecated": false
           }
         }
+      },
+      "targetClientId": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": "11.1"
```

Comment:
This was removed in https://github.com/WebKit/WebKit/commit/f3f63d6b8f665f0d702ab30455778069ab879413, let's not add it.

Reply:
I see this is also set as an alternative name for `replacesClientId` as well.